docs/source/user/settings.rst

```diff
@@ -314,6 +314,32 @@ the configuration should be simply changed to:
         'default': 'https://login.wifiservice.com/{organization}/password/reset/confirm/{uid}/{token}'
     }
 
+
+``OPENWISP_RADIUS_OPTIONAL_REGISTRATION_FIELDS``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Default**: ``{'first_name': 'disabled', 'last_name': 'disabled', 'location': 'disabled', 'birth_date': 'disabled'}``
```

Comment:
can you move the default value on its own line and use a code block (with python formatting) so to break the default value on multiple lines please?